components/webdriver_server/session.rs

```diff
@@ -33,17 +39,109 @@ impl ToString for PageLoadStrategy {
     }
 }
 
+/// Represents the current WebDriver session and holds relevant session state.
+/// Currently, only 1 webview is supported per session.
+/// So only there is only 1 InputState.
+pub struct WebDriverSession {
+    /// <https://www.w3.org/TR/webdriver2/#dfn-session-id>
+    id: Uuid,
+
+    /// <https://www.w3.org/TR/webdriver2/#dfn-current-top-level-browsing-context>
+    /// The id of the current top-level browsing context
+    webview_id: Option<WebViewId>,
+
+    /// <https://www.w3.org/TR/webdriver2/#dfn-current-browsing-context>
+    /// The id of the current browsing context
+    browsing_context_id: Option<BrowsingContextId>,
+
+    timeouts: RefCell<TimeoutsConfiguration>,
```

Comment:
There are some session methods that take &mut already, and there is a session_mut() method to allow calling them. Why do we need to introduce RefCell as well?

Reply:
What's the problem with getting it from the session object?